### DISCUSSION TITLE
Airflow webserver going to hung status without any logs

### DISCUSSION THREAD

**Discussion by venugopal009** [apache/airflow#33466] [General]
### Apache Airflow version

Other Airflow 2 version (please specify below)

### What happened

Airflow version : 2.2.2

Airflow web server went into the hung status and We haven't seen any issue pertaining to scheduler, worker and web server logs.We have checked different metrics published by the Airflow and no issue found on it as well. connectivity to the backend database also working fine.

However the Dag's status is running and task is in the queued state for a longer time, until there is a scheduler, worker restart and it fixed the issue.

### How to reproduce

This issue occurred only once and it is not reproducable.

### Operating System

Linux

### Deployment

Amazon (AWS) MWAA

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by pankajkoti** [comment]
It still looks like some issue with your deployment. Could you try updating your Airflow version to one of the latest releases?
Since there are no reproducible steps and it is an older Airflow version, and it sounds like a troubleshooting question rather than an issue, I am converting this to a discussion.

Please let us know if there are some reproducible steps for this.

cc: @eladkal @shubham22

**Comment by potiuk** [comment]
Yep. Without logs we have 0 chance of doing anything with it, so unless you can find more evidences of what happened, we should keep it as discussion - maybe you will find some more details of your hang. Also what I suggest you is to look closer at your deployment logs and monitoring that you have for your deployment (We do not know what it is, but any kind of deployment contains logs outside of airflow, that airflow has no access to and they usually contain more information on conditions, where you cannot see any logs in airflow - you should look at them in search for mor evidence what happened.

If you find more information - you can post it here and maybe we will try to help you to troubleshoot the problem - but without those evidences, no-one here cann do anything to help to solve your problem and diagnose it.

**Comment by shubham22** [comment]
Agree with others here. Information you've included is insufficient to offer any kind of insight. I would also recommend checking your webserver utilization metrics. If you're heavily reliant on CLI, it could hang webserver as all CLI calls are executed on the webserver. Lastly, recommend upgrading your Airflow version, as things have considerably improved in later versions and it is relatively easy to upgrade with the in-place upgrades, assuming your dependencies/plugin are compatible with Python 3.10.
